### PR TITLE
NAS-100329 doc VMs individual log files.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -350,6 +350,10 @@ These screen options have changed:
 * The :guilabel:`Add to Favorites` checkbox has been removed from
   :menuselection:`Settings --> Preferences`.
 
+* Log files are now generated for each VM. The files are in
```
Comment (Mrt134) [inline userguide/intro.rst]:
```suggestion
* Log files are now generated for each VM and stored in
```

File: userguide/intro.rst [hunk 2]
```diff
@@ -350,6 +350,10 @@ These screen options have changed:
...
+  :file:`/var/log/vm/`. The name of the log file is the name of the
```
Comment (Mrt134) [inline userguide/intro.rst]:
```suggestion
  :file:`/var/log/vm/`. Log files have the same name as the created VM.
```

File: userguide/intro.rst [hunk 3]
```diff
@@ -350,6 +350,10 @@ These screen options have changed:
...
+  VM created.
```
Comment (Mrt134) [inline userguide/intro.rst]:
```suggestion

```

File: userguide/virtualmachines.rst [hunk 4]
```diff
@@ -46,6 +46,10 @@ all of the host CPU.
 :menuselection:`Virtual Machines`
 shows a list of installed virtual machines.
 
+Each VM generates a log file. The files are in
```
Comment (Mrt134) [inline userguide/virtualmachines.rst]:
```suggestion
Each VM generates a log file in :file:`/var/log/vm/`.
```

File: userguide/virtualmachines.rst [hunk 5]
```diff
@@ -46,6 +46,10 @@ all of the host CPU.
...
+:file:`/var/log/vm/`. The name of the log file is the name of the
```
Comment (Mrt134) [inline userguide/virtualmachines.rst]:
```suggestion
The log file has the same name as the VM.
```

File: userguide/virtualmachines.rst [hunk 6]
```diff
@@ -46,6 +46,10 @@ all of the host CPU.
...
+VM created.
```
Comment (Mrt134) [inline userguide/virtualmachines.rst]:
```suggestion

```

File: userguide/intro.rst [hunk 7]
```diff
@@ -358,6 +358,10 @@ These screen options have changed:
 * The :guilabel:`Add to Favorites` checkbox has been removed from
   :menuselection:`Settings --> Preferences`.
 
+
+* Log files are now generated for each VM and stored in
+  :file:`/var/log/vm/`. Log files have the same name as the created VM.
```
Comment (wblock) [inline userguide/intro.rst]: "created" does not mean anything here.

File: userguide/virtualmachines.rst [hunk 8]
```diff
@@ -46,6 +46,9 @@ all of the host CPU.
 :menuselection:`Virtual Machines`
 shows a list of installed virtual machines.
 
+Each VM generates a log file in :file:`/var/log/vm/`.
```
Comment (wblock) [inline userguide/virtualmachines.rst]: VMs don't really "generate" a file...
```suggestion
A log file for each VM is written to :samp:`/var/log/vm/{vmname}`.
```

File: userguide/virtualmachines.rst [hunk 9]
```diff
@@ -46,6 +46,9 @@ all of the host CPU.
...
+The log file has the same name as the VM.
```
Comment (wblock) [inline userguide/virtualmachines.rst]:
```suggestion
```

File: userguide/intro.rst [hunk 10]
```diff
@@ -370,6 +370,10 @@ These screen options have changed:
 * The :guilabel:`Add to Favorites` checkbox has been removed from
   :menuselection:`Settings --> Preferences`.
 
+
+* Log files are now generated for each VM and stored in
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
* Log files for each VM are stored in
```

**Pull request merged**